notify.py

```diff
@@ -34,7 +33,6 @@ def get_service(hass, config, discovery_info=None):
 
     url = config.get(CONF_URL)
     room = config.get(CONF_ROOM)
-    version = int(config.get(CONF_API_VERSION))
 
     try:
         return NextCloudTalkNotificationService(url, username, password, room, version)
```

Comment:
Hey, 
thank you :)
the version should also be removed in NextCloudTalkNotificationService 
and line 56

Reply:
and line 40